nix/shared.nix

```diff
@@ -503,8 +503,20 @@ in
     image-dhall-nixpkgs    = toDockerImage "dhall-nixpkgs"   ;
     image-dhall-yaml       = toDockerImage "dhall-yaml"      ;
 
-    generate-dhall-docs    = pkgs.callPackage ./dhall-docs-gen.nix {
+    prelude-dhall-docs = pkgs.callPackage ./dhall-docs-generator.nix {
       inherit dhall-docs;
+      src = pkgs.fetchurl {
+        url = "https://github.com/dhall-lang/dhall-lang/archive/ccb9f5d54b0ecba05a6493e84442ce445e411e9e.zip";
+        sha256 = "a003e9df45c4bc167d8b65201b0609a7b840cb929c966e7510131b5e50d6bfbe";
+      };
```

Comment:
Why do you need to import the Prelude from another repository?  This repository contains its own copy of the Prelude underneath `./dhall/dhall-lang/Prelude`